### PR TITLE
Copied files should require original if possible

### PR DESCRIPTION
**Pull request by tyler-ball**
@jkeiser's suggested fix for #37.  The compat_resource files are loaded late enough that we _shouldn't_ have dependency or load order issues (because most of the original files will already have been `require`ed).

\cc @lamont-granquist @danielsdeleo @mcquin @jkeiser

### CODE REVIEW

**Comment by danielsdeleo** [comment]
:+1:

File: Rakefile [hunk 1]
```diff
@@ -89,6 +89,11 @@ task :update do
   chef_gem_path = Bundler.environment.specs['chef'].first.full_gem_path
   CHEF_FILES.each do |file|
     output = StringIO.new
+    # First lets try to load the original file if it exists
+    output.puts "begin"
+    output.puts "  require '#{file}'"
+    output.puts "rescue LoadError; end"
+    output.puts ""
```
Comment (mcquin) [inline Rakefile]: debugging?

**Comment by mcquin** [comment]
:rocket:

**Comment by mcquin** [comment]
Is a version bump needed?

**Comment by lamont-granquist** [comment]
:+1:

**Comment by tyler-ball** [comment]
@mcquin I'm sure one is, but I don't have access to merge this PR or to push the cookbook

**Comment by afiune** [comment]
:shipit:

**Comment by oferrigni** [comment]
@tyler-ball Can you bump the version in the metadata?

**Comment by oferrigni** [comment]
This fix has been tested/validated in shd and works. Once metadata is bumped, should be good to merge.

**Pull request merged**